example-config.json

```diff
@@ -30,7 +30,12 @@
         "port":51233
     },
     "log_level":"debug",
-    "log_file":"./clio.log",
+    "log_to_console": true,
+    "log_to_file": true,
+    "log_rotation_size": 2147483648,
```

Comment:
How hard would it be to be able to specify that in GB

Like
```json
"log_rotation_size": "2GB" // "2MB"
```

If it's hard, don't worry about it, but it would be way more readable.